game_starter.py

```diff
@@ -67,8 +76,21 @@ def get_guessed_word(secret_word, letters_guessed):
     returns: string, comprised of letters and underscores that represents
       what letters in secret_word have been guessed so far.
     '''
-    # FILL IN YOUR CODE HERE...
-    pass
+    '''
+    for every letter in secret word
+      if letter is in guess letter
+        print the letter
+      if not
+        print _
+    '''
+    word = ""
+    for letter in secret_word:
+      if letter in letters_guessed:
+        word += letter
+      else:
+        word += '_'
```

Comment:
When inserting underscores into your string, it's a good idea to add at least a space after each 
one, so it's clear to the user how many unguessed letters are left in the string (compare the 
readability of 
____
with 
_ _ _ _ 
).